js/common.js

```diff
@@ -0,0 +1,8 @@
+const $ = (select, el = document) => el.querySelector(select);
```

Comment:
이렇게 매개변수가 표현되는게 함수를 이해하는 데 있어서 더욱 명확할 것 같아요!
```suggestion
const $ = (selector, element = document) => element.querySelector(selector);
```